src/echo/echo.c

```diff
@@ -0,0 +1,78 @@
+/*
+** EPITECH PROJECT, 2023
+** echo.c
+** File description:
+** for 42sh
+*/
+
+#include "mysh.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+char *adapt_str(char *str, int nb_quotes)
+{
+    int size = strlen(str) - nb_quotes;
+    char *new_str = malloc(sizeof(char) * size + 1);
```

Comment:
tu peux vérifier le malloc et si t'as besoin de plus de ligne tu peux mettre le "j++" de la ligne 30 dans le for =)